packages/api/bin/serve.js

```diff
@@ -127,74 +140,90 @@ function checkEnvVariablesAreSet(moreRequiredEnvVars) {
-  const collection = await cm.create(c);
-  await indexer.indexCollection(esClient, collection, esIndex);
+  const collection = testUtils.fakeCollectionFactory({ name: `${stackName}-collection` });
+  await serveUtils.addCollections([collection], esClient, esIndex);
 
   // add granule records
-  const g = testUtils.fakeGranuleFactory();
-  g.granuleId = `${stackName}-granule`;
-  const gm = new models.Granule();
-  const granule = await gm.create(g);
-  await indexer.indexGranule(esClient, granule, esIndex);
+  const granule = testUtils.fakeGranuleFactoryV2({ granuleId: `${stackName}-granule` });
+  await serveUtils.addGranules([granule], esClient, esIndex);
 
   // add provider records
-  const p = testUtils.fakeProviderFactory();
-  p.id = `${stackName}-provider`;
-  const pm = new models.Provider();
-  const provider = await pm.create(p);
-  await indexer.indexProvider(esClient, provider, esIndex);
+  const provider = testUtils.fakeProviderFactory({ id: `${stackName}-provider` });
+  await serveUtils.addProviders([provider], esClient, esIndex);
 
   // add rule records
-  const r = testUtils.fakeRuleFactoryV2();
-  r.name = `${stackName}_rule`;
-  r.workflow = workflowList[0].name;
-  r.provider = `${stackName}-provider`;
-  r.collection = {
-    name: `${stackName}-collection`,
-    version: '0.0.0'
-  };
-  const rm = new models.Rule();
-  const rule = await rm.create(r);
-  await indexer.indexRule(esClient, rule, esIndex);
+  const rule = testUtils.fakeRuleFactoryV2({
+    name: `${stackName}_rule`,
+    workflow: workflowList[0].name,
+    provider: `${stackName}-provider`,
+    collection: {
+      name: `${stackName}-collection`,
+      version: '0.0.0'
+    }
+  });
+  await serveUtils.addRules([rule], esClient, esIndex);
 
   // add fake execution records
-  const e = testUtils.fakeExecutionFactory();
-  e.arn = `${stackName}-fake-arn`;
-  const em = new models.Execution();
-  const execution = await em.create(e);
-  await indexer.indexExecution(esClient, execution, esIndex);
+  const execution = testUtils.fakeExecutionFactoryV2({ arn: `${stackName}-fake-arn` });
+  await serveUtils.addExecutions([execution], esClient, esIndex);
 
   // add pdrs records
-  const pd = testUtils.fakePdrFactory();
-  pd.pdrName = `${stackName}-pdr`;
-  const pdm = new models.Pdr();
-  await pdm.create(pd);
+  const pdr = testUtils.fakePdrFactoryV2({ pdrName: `${stackName}-pdr` });
+  await serveUtils.addPdrs([pdr], esClient, esIndex);
 }
 
 /**
  * Prepare and run the Cumulus API Express app.
  *
  * @param {string} user - A username to add as an authorized user for the API.
  * @param {string} stackName - The name of local stack. Used to prefix stack resources.
+ * @param {bool} reseed - boolean to control whether to load new data into
+ *                        dynamo and elastic search.
  */
-async function serveApi(user, stackName = defaultLocalStackName) {
+async function serveApi(user, stackName, reseed = true) {
```

Comment:
I would restore the use of `defaultLocalStackName` here after removing the default from the `bin/cli.js` file